simpler_env/main_inference.py

```diff
@@ -7,6 +7,7 @@
 from simpler_env.evaluation.maniskill2_evaluator import maniskill2_evaluator
 from simpler_env.policies.octo.octo_server_model import OctoServerInference
 from simpler_env.policies.rt1.rt1_model import RT1Inference
+from simpler_env.policies.openvla.openvla_model import OpenVALInference
```

Comment:
OpenVALInference should be OpenVLAInference?

Reply:
This code is outdated.